cocos/core/renderer/scene/camera.ts

```diff
@@ -226,10 +226,13 @@ export class Camera {
     }
 
     public destroy () {
+        this.detachFromScene();
```

Comment:
If not detach from scene, camera will hold scene, and it is put to CameraPool, then scene will not be destroyed.